pinot-clients/pinot-java-client/src/main/java/org/apache/pinot/client/JsonAsyncHttpPinotClientTransport.java

```diff
@@ -66,7 +66,8 @@ public JsonAsyncHttpPinotClientTransport() {
   }
 
   public JsonAsyncHttpPinotClientTransport(Map<String, String> headers, String scheme,
-    @Nullable SSLContext sslContext, ConnectionTimeouts connectionTimeouts, TlsProtocols tlsProtocols) {
+                                           @Nullable SSLContext sslContext, ConnectionTimeouts connectionTimeouts,
```

Comment:
(format) Please follow the [Pinot Style](https://docs.pinot.apache.org/developers/developers-and-contributors/code-setup#setup-ide). Same for other changes

Reply:
Annotate `appId` as `@Nullable`

Reply:
The format is still not correct. Please update the IDE style setting and reformat